Sorts/FindSecondLargestElement.js

```diff
@@ -11,7 +11,7 @@
 
 const secondLargestElement = (array) => {
   const largestElement = Math.max(...array)
```

Comment:
This is a problem of the original algorithm, but shouldn't this be done in the loop? This way of solving it seems like cheating ¯\\\_(ツ)\_/¯

Reply:
> This is a problem of the original algorithm, but shouldn't this be done in the loop? This way of solving it seems like cheating ¯\_(ツ)_/¯

Nah, it's fine; this is supposed to be idiomatic JS. Otherwise a `findMax` algo should probably be required, or `secondLargestElement` should maintain the two largest elements at a time.

TBF using a heap this may be generalized to findNthLargestElement...